### PR TITLE
Localizes stock text.

### PR DESCRIPTION
**Pull request by harmonyn**
I finally did it goof

probably gonna add to this, I want to write more events

Also: Stock logs show share price at time of purchase/sale because GODDAMN GOOF HOW DO YOU CODE A LOG WITHOUT THE MOST IMPORTANT PART

### CODE REVIEW

**Comment by optimumtact** [comment]
:-1: use the strings system like they do for ai ion laws

File: code/modules/stock_market/events.dm [hunk 1]
```diff
@@ -160,12 +160,12 @@
 	else
 		position = ucfirsts(company.industry.detokenize("Lead %industrial% Engineer"))
 	offenses = ""
-	var/list/O = list("corruption", "murder", "jaywalking", "assault", "battery", "drug possession", "burglary", "theft", "larceny", "bribery",
```
Comment (Cheridan) [inline code/modules/stock_market/events.dm]: >removing jaywalking

class 17 felony

Comment (optimumtact) [inline code/modules/stock_market/events.dm]: yeah this is funny keep it in

File: code/modules/stock_market/stockmarket.dm [hunk 2]
```diff
@@ -60,10 +60,10 @@
 		return d
 
 /datum/stockMarket/proc/generateStocks(var/amt = 15)
-	var/list/fruits = list("Banana", "Strawberry", "Watermelon", "Maracuja", "Pomegranate", "Papaya", "Mango", "Tomato", "Conkerberry", "Fig", "Lychee", "Mandarin", "Oroblanco", "Pumpkin", "Rhubarb", "Tamarillo", "Yantok", "Ziziphus", "Oranges")
+	var/list/fruits = list("Banana", "Mimana", "Watermelon", "Ambrosia", "Pomegranate", "Daisy", "Papaya", "Mango", "Tomato", "Conkerberry", "Wood", "Lychee", "Mandarin", "Harebell", "Pumpkin", "Rhubarb", "Tamarillo", "Yantok", "Ziziphus", "Oranges", "Gatfruit", "Daisy", "Kudzu")
```
Comment (Cheridan) [inline code/modules/stock_market/stockmarket.dm]: Daisy is in the list twice

File: code/modules/stock_market/stockmarket.dm [hunk 3]
```diff
@@ -60,10 +60,10 @@
...
 	var/list/tech_prefix = list("Nano", "Cyber", "Funk", "Astro", "Fusion", "Tera", "Exo", "Star", "Virtual", "Plasma", "Robust", "Bit", "Future", "Hugbox", "Carbon", "Nerf", "Buff", "Nova", "Space")
 	var/list/tech_short = list("soft", "tech", "prog", "tec", "tek", "ware", "", "gadgets", "nics", "tric", "trasen", "tronic", "coin")
-	var/list/random_nouns = list("Johnson", "Cluwne", "General", "Specific", "Master", "King", "Queen", "Wayne", "Rupture", "Dynamic", "Massive", "Mega", "Giga", "Certain", "Stale", "State", "National", "International", "Interplanetary", "Sector", "Planet", "Burn", "Robust", "Exotic", "Solar", "Cheesecake", "Chelp", "Corgi")
+	var/list/random_nouns = list("Johnson", "Cluwne", "General", "Specific", "Master", "King", "Queen", "Table", "Rupture", "Dynamic", "Massive", "Mega", "Giga", "Certain", "Singulo", "State", "National", "International", "Interplanetary", "Sector", "Planet", "Burn", "Robust", "Exotic", "Solar", "Lunar", "Chelp", "Corgi", "Lag")
```
Comment (Cheridan) [inline code/modules/stock_market/stockmarket.dm]: sorry tgstation is 100% lag free

File: code/modules/stock_market/articles.dm [hunk 4]
```diff
@@ -33,27 +33,27 @@ proc/list_frozen()
...
-		"development" = list("development", "unfolding of events", "turn of events"), \
-		"dip" = list("dip", "fall", "rapidly descend", "decrease"), \
+		"country" = list("Space", "Argentina", "Hungary", "United States of America", "United Space", "Space Federation", "Nanotrasen", "The Wizard Federation", "United Kingdom", "Poland", "The Syndicate", "Australia", "Serbia", "The European Union", "The Illuminati", "The New World Order", "Eurasian Union", "Asian Union", "United Arab Emirates", "Arabian League", "United States of Africa", "Mars Federation", "Allied Colonies of Jupiter", "Saturn's Ring", "Fringe Republic of Formerly Planet Pluto"), \
+		"development" = list("development", "unfolding of events", "turn of events", "new shit"), \
+		"dip" = list("dip", "fall", "plunge", "decrease"), \
 		"excited" = list("excited", "euphoric", "exhilarated", "thrilled", "stimulated"), \
-		"expand_influence" = list("expands their influence over", "continues to dominate", "looks to gain shares in", "rolls their new product line out in"), \
-		"failure" = list("failure", "meltdown", "breakdown", "crash", "defeat", "trainwreck", "wreck"), \
```
Comment (KazeEspada) [inline code/modules/stock_market/articles.dm]: Keep trainwreck

File: code/modules/stock_market/articles.dm [hunk 5]
```diff
@@ -81,7 +81,7 @@ proc/list_frozen()
 	ticks = world.time
 
 /datum/article/proc/generateOutletName()
-	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia")
+	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia", "Orion", "Sybil", "Basil", "Badger", "Terry", "Artyom")
```
Comment (KazeEspada) [inline code/modules/stock_market/articles.dm]: Remove Pluto?

File: code/modules/stock_market/stockmarket.dm [hunk 6]
```diff
@@ -22,7 +22,7 @@
 	stockBrokers = list()
 	var/list/fnames = list("Goldman", "Edward", "James", "Luis", "Alexander", "Walter", "Eugene", "Mary", "Morgan", "Jane", "Elizabeth", "Xavier", "Hayden", "Samuel", "Lee")
 	var/list/names = list("Johnson", "Rothschild", "Sachs", "Stanley", "Hepburn", "Brown", "McColl", "Fischer", "Edwards", "Becker", "Witter", "Walker", "Lambert", "Smith", "Montgomery", "Lynch", "Roosevelt", "Lehman")
-	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia")
+	var/list/locations = list("Earth", "Luna", "Mars", "Saturn", "Jupiter", "Uranus", "Pluto", "Europa", "Io", "Phobos", "Deimos", "Space", "Venus", "Neptune", "Mercury", "Kalliope", "Ganymede", "Callisto", "Amalthea", "Himalia", "Sybil", "Basil", "Badger", "Terry", "Artyom")
```
Comment (KazeEspada) [inline code/modules/stock_market/stockmarket.dm]: Remove Pluto?

File: code/modules/stock_market/articles.dm (hunk 5 above)
Comment (bgobandit) [inline code/modules/stock_market/articles.dm]: pluto is and always will be a planet fam

File: code/modules/stock_market/industries.dm [hunk 7]
```diff
@@ -153,13 +153,13 @@
...
+		"jobs" = list("medical doctors", "nurses", "paramedics", "psychologists", "psychiatrists")
 	)
 
 /datum/industry/health/generateProductName(var/company_name)
-	var/list/prefix = list("amino", "nucleo", "nitro", "panto", "meth", "eth", "as", "algo", "coca", "hero", "morph", "trinitro", "prop", "but", "acet", "acyclo", "lansop", "dyclo", "hydro", "oxycod", "vicod")
-	var/list/suffix = list("phen", "pirin", "pyrine", "ane", "amphetamine", "prazoline", "ine", "yl", "amine", "aminophen", "one", "ide", "phenate", "anol", "toulene", "glycerine", "vir")
-	var/list/uses = list("antidepressant", "analgesic", "anesthetic", "antiretroviral", "antiviral", "antibiotic", "cough drop", "depressant", "hangover cure", "homeopathic", "fertility drug", "hypnotic", "narcotic", "laxative", "multivitamin", "purgative", "relaxant", "steroid", "sleeping pill", "suppository", "traquilizer")
+	var/list/prefix = list("amino", "nucleo", "nitro", "panto", "meth", "eth", "as", "algo", "coca", "hero", "lotsu", "morph", "trinitro", "prop", "but", "acet", "acyclo", "lansop", "dyclo", "hydro", "oxycod", "vicod", "cryo", "dex", "chloro")
```
Comment (Niknakflak) [inline code/modules/stock_market/industries.dm]: busted

Comment (Aloraydrel) [inline code/modules/stock_market/industries.dm]: >Not owning over 30 collectable lotsu dolls

Look at this casual

Comment (bgobandit) [inline code/modules/stock_market/industries.dm]: it's a prefix

Comment (Cheridan) [inline code/modules/stock_market/industries.dm]: what does the prefix indicate

File: code/modules/stock_market/industries.dm [hunk 8]
```diff
@@ -75,7 +75,7 @@
 								"%company_name% looking to reap profits off the %industrial% sector with %product_name%")
 
 /datum/industry/agriculture/generateProductName(var/company_name)
-	var/list/products = list("combine harvester", "cattle prod", "scythe", "plough", "sickle", "cloche", "loy", "spade", "hoe")
```
Comment (Iamgoofball) [inline code/modules/stock_market/industries.dm]: combine harvesters are real IRL farming machines

File: code/modules/stock_market/industries.dm [hunk 9]
```diff
@@ -179,11 +179,14 @@
...
-		"jobs" = list("miners", "drill operators", "mining foremen", "explosives handlers")
+		"industrial" = list("resource acquisitional"), \
+		"jobs" = list("shaft miners", "drill operators", "mining foremen", "gibtonite handlers")
 	)
 
 /datum/industry/mining/generateProductName(var/company)
-	var/list/equipment = list("drill", "pickaxe", "shovel", "mini-pickaxe", "power hammer", "power gloves", "power armor", "hardsuit", "oxygen tank", "emergency bike horn")
-	var/list/material = list("mauxite", "pharosium", "molitz", "char", "ice", "cobryl", "bohrum", "claretine", "viscerite", "syreline", "cerenkite", "plasmastone", "gold", "koshmarite")
+	var/list/equipment = list("drill", "pickaxe", "shovel", "jackhammer", "mini-pickaxe", "power hammer", "power gloves", "power armor", "hardsuit", "kinetic accelerator", "resonator", "oxygen tank", "emergency bike horn")
+	var/list/material = list("mauxite", "pharosium", "molitz", "adamantium", "mithril", "cobryl", "bohrum", "claretine", "viscerite", "syreline", "cerenkite", "plasmastone", "gold", "koshmarite", "phoron", "carbon dioxide")
```
Comment (Iamgoofball) [inline code/modules/stock_market/industries.dm]: >>>PHORON

File: code/modules/stock_market/events.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/stock_market/events.dm]: agreed, put jaywalking back in

**Comment by Iamgoofball** [comment]
:+1:

File: code/modules/stock_market/industries.dm (hunk 8 above)
Comment (bgobandit) [inline code/modules/stock_market/industries.dm]: I know but some shit has to go

Comment (Iamgoofball) [inline code/modules/stock_market/industries.dm]: Why?

Comment (bgobandit) [inline code/modules/stock_market/industries.dm]: so this shit does not become the size of the entire game

File: code/modules/stock_market/industries.dm (hunk 9 above)
Comment (duncathan) [inline code/modules/stock_market/industries.dm]: i mean it IS a foreign substance

File: code/modules/stock_market/industries.dm (hunk 8 above)
Comment (Iamgoofball) [inline code/modules/stock_market/industries.dm]: That's not a reason???
@cheridan

File: code/modules/stock_market/industries.dm (hunk 7 above)
Comment (bgobandit) [inline code/modules/stock_market/industries.dm]: it is of ghanaian origin and indicates concentration or purity

File: code/modules/stock_market/industries.dm (hunk 8 above)
Comment (Cheridan) [inline code/modules/stock_market/industries.dm]: sorry goffball but we're removing your stupid half life 2 memes

**Comment by optimumtact** [comment]
:-1: still not using either text files or json strings

**Comment by bgobandit** [comment]
merge when

**Comment by KorPhaeron** [comment]
@bgobandit 

When you change the stuff Cheridan said to change

**Comment by Cheridan** [comment]
It's ok

**Comment by bgobandit** [comment]
I already put jaywalking in it's just not showing up for some reason

**Pull request merged**